peglib.h

```diff
@@ -956,6 +957,21 @@ class Ope
     virtual ~Ope() {}
     virtual size_t parse(const char* s, size_t n, SemanticValues& sv, Context& c, any& dt) const = 0;
     virtual void accept(Visitor& v) = 0;
+protected:
+    void traceResult(bool match, size_t cnt, const char *name, const char *s,
+                     size_t n, SemanticValues& sv, Context& c, any& dt) const
+    {
+        if (c.tracer) {
+            std::stringstream str;
+            if (match) {
+                str << "*" << name << " matched " << std::to_string(cnt) << "*";
```

Comment:
It may not be a good idea to include a specific trace output format in peglib.h, since each application of peglib may want a different output format.